cloud/azure/module-integration.tf

```diff
@@ -1,4 +1,4 @@
-module "sfx-integration" {
+module "sfx_integration" {
```

Comment:
I am afraid this will lead to recreation when using the next version (except if user manually state mv resources)

Reply:
Yes, but moving to AzureAD v2 is also a breaking change, so I prefered to follow tflint code rules for module and resource naming.

Reply:
as you want